rdbms/src/test/java/org/spine3/server/storage/jdbc/stand/JdbcStandStorageShould.java

```diff
@@ -20,39 +20,66 @@
 
 package org.spine3.server.storage.jdbc.stand;
 
+import com.google.common.base.Optional;
 import com.google.protobuf.Descriptors;
 import com.google.protobuf.FieldMask;
 import com.google.protobuf.Message;
 import org.junit.Test;
 import org.spine3.protobuf.AnyPacker;
-import org.spine3.protobuf.Timestamps;
+import org.spine3.protobuf.Timestamps2;
 import org.spine3.protobuf.TypeUrl;
 import org.spine3.server.aggregate.Aggregate;
+import org.spine3.server.entity.AbstractEntity;
+import org.spine3.server.entity.EntityRecord;
 import org.spine3.server.stand.AggregateStateId;
 import org.spine3.server.stand.StandStorage;
-import org.spine3.server.storage.EntityStorageRecord;
+import org.spine3.server.stand.StandStorageShould;
 import org.spine3.server.storage.jdbc.DatabaseException;
 import org.spine3.server.storage.jdbc.GivenDataSource;
 import org.spine3.server.storage.jdbc.JdbcStandStorage;
 import org.spine3.server.storage.jdbc.entity.query.CreateEntityTableQuery;
-import org.spine3.server.storage.jdbc.entity.query.EntityStorageQueryFactory;
+import org.spine3.server.storage.jdbc.entity.query.RecordStorageQueryFactory;
+import org.spine3.server.storage.jdbc.entity.visibility.query.CreateVisibilityTableQuery;
 import org.spine3.server.storage.jdbc.util.DataSourceWrapper;
-import org.spine3.test.aggregate.Project;
-import org.spine3.test.aggregate.ProjectId;
 import org.spine3.test.commandservice.customer.Customer;
+import org.spine3.test.storage.Project;
+import org.spine3.test.storage.ProjectId;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
 
 import static junit.framework.TestCase.assertEquals;
-import static org.junit.Assert.*;
-import static org.mockito.Mockito.*;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 import static org.spine3.test.Verify.assertContains;
 import static org.spine3.test.Verify.assertSize;
 
 /**
  * @author Dmytro Dashenkov
  */
-public class JdbcStandStorageShould {
+public class JdbcStandStorageShould extends StandStorageShould {
+
+    @Override
+    protected StandStorage getStorage() {
+        final DataSourceWrapper dataSource = GivenDataSource.whichIsStoredInMemory(
+                "StandStorageTests");
+        return JdbcStandStorage.newBuilder()
+                               .setDataSource(dataSource)
+                               .setRecordStorageQueryFactory(
+                                       new RecordStorageQueryFactory<>(dataSource,
```

Comment:
Why not extract a variable?

The same applies to the whole `return` expression.